src/modules/views/Folder/SharedDrives.tsx

```diff
@@ -51,7 +48,7 @@ export const ExtraDrive = ({
                 onFileOpen={handleFileOpen}
                 isInSyncFromSharing={false}
                 disableSelection={true}
-                isExternalDrive={true}
+                isSharedDrives={true}
```

Comment:
This is where the subtlety comes in. This component knows that it is displaying the contents of shared drives. But we specify the display cases for the children. Here an external drive and in the future a shared drive.

Reply:
yes, instead I will check if it's of type shortcut. I assume shared drives will be of type file or directory?

Reply:
From my understanding it would be a directory